config/locales/en.yml

```diff
@@ -2567,6 +2567,10 @@ en:
             body_html: |
               <p>By using a computer system and software to prepare and file my tax return(s) electronically, I consent to the transmission of my return(s) and to the disclosure of all information about my use of the system and software to the North Carolina Department of Revenue.</p>
               <p>By checking the box below, I declare and certify that I have examined this return and accompanying schedules and statements, and to the best of my knowledge and belief, they are true, correct, and complete.</p>
+          nj:
+            body_html: |
+              <p>Under penalties of perjury, I declare that I have examined this Income Tax return, including accompanying schedules and statements, and to the best of my knowledge and belief, it is true, correct, and complete. (N.J.S.A. 2C:28-1)</p>
```

Comment:
[dust] More of a content question but should (N.J.S.A. 2C:28-1) be a link to somewhere? Or should we explain somewhere what this is? Thinking of [WCAG level AAA abbreviation requirement.](https://www.w3.org/TR/WCAG22/#abbreviations)

Reply:
I agree! Jamie has yet to do a final content pass on this though, so I think that change will be coming later. Ticket was mainly to get the logic implemented to separate out NJ and AZ esignature statements.